### PR TITLE
Lra 176 remove sla agreements

### PR DESCRIPTION
**Pull request by britaumich**
I removed sla_agreement and sla attachment from views, CSV export, and active admin.
I removed sla attachment from the model but left the sla_agreement field in the model.

### CODE REVIEW

**Review by rsmoke — CHANGES_REQUESTED** [review]
Please delete the route instead of commenting it. Did your model test run without issue?

File: config/routes.rb [hunk 1]
```diff
@@ -33,7 +33,7 @@
   end
 
   get 'dpa_exceptions/audit_log/:id', to: 'dpa_exceptions#audit_log', as: :dpa_exception_audit_log
-  get 'dpa_exceptions/delete_file_attachment/:id', to: 'dpa_exceptions#delete_file_attachment', as: :delete_sla_agreement
+  # get 'dpa_exceptions/delete_file_attachment/:id', to: 'dpa_exceptions#delete_file_attachment', as: :delete_sla_agreement
```
[the remark this answers is not in the document]
Comment (britaumich) [reply config/routes.rb]: The commented route is deleted.

**Comment by britaumich** [comment]
Yes, models specs run, because sla_agreement was not required for a record to be complete.

File: config/routes.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (rsmoke) [reply config/routes.rb]: 👍

**Review by rsmoke — APPROVED** [review]
Great

**Pull request merged**